demo/ghostExample/PlaygroundGhostExample.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import type {StoryFn} from '@storybook/react';
+
+import {PlaygroundGhostExample} from './PlaygroundGhostExample';
+
+export default {
+    title: 'Markdown Editor / Docs example',
```

Comment:
Let’s place it in the “Experiments” section.
We will name the page “Popup in Markup Mode”.